Dockerfile

```diff
@@ -23,13 +23,13 @@ RUN dnf install -y --setopt=tsflags=nodocs \
     && dnf clean all
 
 # Install webdrivers
-RUN GECKODRIVER_VERSION="v0.26.0" && \
+RUN GECKODRIVER_VERSION="v0.27.0" && \
     wget https://github.com/mozilla/geckodriver/releases/download/$GECKODRIVER_VERSION/geckodriver-$GECKODRIVER_VERSION-linux64.tar.gz && \
     tar -zxf geckodriver-$GECKODRIVER_VERSION-linux64.tar.gz -C /usr/local/bin && \
     chmod +x /usr/local/bin/geckodriver && \
     rm geckodriver-$GECKODRIVER_VERSION-linux64.tar.gz
 
-RUN CHROMEDRIVER_VERSION="80.0.3987.106" && \
+RUN CHROMEDRIVER_VERSION="86.0.4240.22" && \
```

Comment:
Is this version compatible w/ the chromium version?

Reply:
Yes compatible and tested